kuma/static/js/wiki-compat-trigger.js

```diff
@@ -0,0 +1,48 @@
+(function(win, doc, $) {
+    'use strict';
+
+    /*
+        Load and build compat tables if present in page
+    */
+    (function() {
+        var compatCSS;
+        var compatJS;
+
+        // don't run if no compat table on page with min 1 row
+        var $compatFeatureRows = $('.bc-table tbody tr');
+        if(!$compatFeatureRows.length) {
+            // TODO return;
+        }
+
+        // don't run if waffle not active
+        if(!win.waffle || !win.waffle.flag_is_active('compat_data')) {
+            return;
+        }
+
+        // don't run if assests not available
+        if(!win.mdn || !mdn.assets) {
+            return;
+        }
```

Comment:
Would it make sense to move lines 11 - 15 to below the above? If the waffle flag is not active, or the assets are not available, we do not need to bother with line 11 - 15, correct?

Reply:
I moved the waffle check up but the compat table is more likely to be missing than the assets (it's on very few pages right now)